lib/models/builder.js

```diff
@@ -215,6 +215,9 @@ class Builder extends CoreObject {
       .then(result => {
         self.project._instrumentation.stopAndReport('build', result, resultAnnotation);
         return result;
+      }, reason => {
+        self.project._instrumentation.stopAndReport('build', null, resultAnnotation);
```

Comment:
@stefanpenner i added this as well as `_buildSummary` tolerance for lack of result to ensure we don't mask an error here.

I was getting a local error b/c I had symlinked `ember-cli` and had an out of date `broccoli-babel-transpiler` (5.6.0 instead of 5.6.2).  In any case, seems good to not mask errors.